src/main/java/com/openshift/jenkins/plugins/pipeline/OpenShiftBuilder.java

```diff
@@ -69,15 +72,33 @@ public String getCheckForTriggeredDeployments() {
 	public String getWaitTime() {
 		return waitTime;
 	}
+
+	public String getWaitUnits() { return waitUnit; }
 	
 	public String getWaitTime(Map<String,String> overrides) {
 		String val = getOverride(getWaitTime(), overrides);
 		if (val.length() > 0)
 			return val;
 		return Long.toString(getDescriptor().getWait());
 	}
-	
-	// Overridden for better type safety.
+
+    @Override
+    public String getWaitTimeMilli(Map<String, String> overrides) {
```

Comment:
You should be able to move this to IOpenShiftBuilder as a default method, and then share the implementation between the dsl and classic freestyle versions